### PR TITLE
Add a news item for the 2022 financial report

### PR DESCRIPTION
**Pull request by leavengood**

### CODE REVIEW

File: content/news/2023-01-22_2022_financial_report.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
++++
+
+The Haiku, Inc. [financial report for 2022](https://www.haiku-inc.org/docs/haiku_inc-financial-report-2022.pdf)
+is now available on the [Haiku, Inc. Documents page](https://www.haiku-inc.org/documents/).
+
+Our donations for 2022 were almost as high as 2021, which was our record year so
+far.
+
+In 2022 our contractor waddlesplash worked the whole year, and we got a Beta 4
+release near Christmas due to his work as well as great work from many other contributors.
```
Comment (humdingerb) [inline content/news/2023-01-22_2022_financial_report.md:16]: A bt much "work" went into this sentence. :)

```suggestion
release near Christmas due to his efforts as well as from our many other contributors.
```

File: content/news/2023-01-22_2022_financial_report.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+I want to extend a huge thanks to all our donors, everyone in the project really
+appreciates every donation we get. A lot got done in 2022 and 2023 should be another
+great year for Haiku. I won't make any predictions as far as releases are concerned
+(I've learned not to do that anymore), but I know Haiku will continue to improve
+and become a better system for daily use every month. Much of this is due to the
+support of our donors and that we can pay someone to work on Haiku.
```
Comment (humdingerb) [inline content/news/2023-01-22_2022_financial_report.md:23]:
```suggestion
support of our donors which allows us to pay someone to work on Haiku.
```

**Comment by leavengood** [comment]
Thanks for the edits humdinger, that's why I put this up for review 😄 

I'll give it another few hours for everyone else to look, then we can merge.

**Comment by pulkomandy** [comment]
no extra comments from me. Thanks for the report :)

**Pull request merged**